net.solarnetwork.node.weather.ibm.wc/src/net/solarnetwork/node/weather/ibm/wc/HourlyDatumPeriod.java

```diff
@@ -0,0 +1,38 @@
+
+package net.solarnetwork.node.weather.ibm.wc;
+
+public enum HourlyDatumPeriod implements DatumPeriod {
+
+	SIXHOUR("6hour"),
+	TWELVEHOUR("12hour"),
+	ONEDAY("1day"),
+	TWODAY("2day"),
+	THREEDAY("3day"),
+	TENDAY("10day"),
+	FIFTEENDAY("15day");
+
+	private String period;
+
+	HourlyDatumPeriod(String period) {
+		this.period = period;
+	}
+
+	public String getPeriod() {
+		return this.period;
+	}
+
+	@Override
+	public String toString() {
+		return this.getPeriod();
+	}
+
+	static HourlyDatumPeriod getValue(String period) {
+		for ( HourlyDatumPeriod e : HourlyDatumPeriod.values() ) {
+			if ( e.getPeriod() == period ) {
```

Comment:
Comparing strings with `==` is probably not correct, maybe you mean

```java
if ( e.getPeriod.equals(period) ) {
```

or I'd say it would even be useful to ignore case with

```java
if ( e.getPeriod().equalsIgnoreCase(period) ) {
```